staging/inputs.nix

```diff
@@ -0,0 +1,14 @@
+# This gives Flakes vibes for everyone who are missing them.
+{ ... }:
+let
+  sources = import ./npins;
+in
+{
+  imports = [
+    "${sources.agenix}/modules/age.nix"
+  ];
+
+  _module.args.inputs = {
+    inherit sources;
+  };
```

Comment:
I think this should just go into `configuration.nix`, I found it needlessly hard to follow.

Reply:
That's fine to me.